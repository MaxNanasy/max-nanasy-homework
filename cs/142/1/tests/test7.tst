number ident letter parameters ERROR