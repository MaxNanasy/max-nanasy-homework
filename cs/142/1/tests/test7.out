 ident  ident  ident  ident  ident  <EOT>